docs/dxp/latest/en/system-administration/file-storage/other-file-store-types.md

```diff
@@ -0,0 +1,20 @@
+# Other File Store Types
+
+```{toctree}
+:maxdepth: 3
+
+   other-file-store-types/simple-file-system-store.md
+   other-file-store-types/amazon-s3-store.md
+   other-file-store-types/dbstore.md
+   other-file-store-types/ibm-cloud-object-storage.md
+   other-file-store-types/google-cloud-storage.md
+   other-file-store-types/microsoft-azure-blob-storage.md
```

Comment:
@Fabiomorais87  Please remove the indents from these toctree entries. MyST expects them to start at the beginning of each line. As such, they're not showing in the left nav when you're in the Other File Store Types section or in any of its articles.

Thanks.